tests/integration/partnerOrganizations.test.js

```diff
@@ -232,9 +221,9 @@ describe('PARTNER ORGANIZATION ROUTES - GET /partnerorganizations/{_id}', () =>
 
 describe('PARTNER ORGANIZATION ROUTES - PUT /partnerorganizations/{_id}', () => {
   let authToken;
-  beforeEach(populateDB);
 
   describe('COACH', () => {
+    beforeEach(populateDB);
     beforeEach(async () => {
       authToken = await getToken('coach');
     });
```

Comment:
same pour 'should update a partner organization'

Reply:
done